contracts/RootChain/RootChain.sol

```diff
@@ -106,33 +98,40 @@ contract RootChain {
         public
         payable
     {
-        require(blocknum == validatorBlocks);
+        require(currentDepositBlock < childBlockInterval);
+        require(blocknum == currentChildBlock);
         var txList = txBytes.toRLPItem().toList();
-        require(txList.length == 11);
-        for(uint256 i = 0; i < 6; i++) {
+        require(txList.length == 15);
```

Comment:
Might want to update comments/specify the 15 items in the decoded form are

Reply:
Updated deposit documentation to include transaction encoding